src/itest/resources/application.yml

```diff
@@ -21,4 +21,6 @@ api:
   internal-api-url: ${INTERNAL_API_URL:localhost}
 
 logger:
-  namespace: psc-delta-consumer
\ No newline at end of file
+  namespace: psc-delta-consumer
+
```

Comment:
I'm not sure we should commit this debug line, I don't think it actually fixed anything right and it's VSCode specific